app/assets/stylesheets/dashboard/ssr_print.css.scss

```diff
@@ -0,0 +1,600 @@
+// Copyright © 2011 MUSC Foundation for Research Development
+// All rights reserved.
+
+// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+
+// 1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+
+// 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following
+// disclaimer in the documentation and/or other materials provided with the distribution.
+
+// 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products
+// derived from this software without specific prior written permission.
+
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING,
+// BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+// SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+// TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+@media print {
+  /* HIDING ELEMENTS */
+  button, select, .bootstrap-select {
+    display: none;
+  }
+  /* END HIDING ELEMENT */
+
+
+
+  /* GENERAL STUFF */
+  .text-center {
+    text-align: center;
+  }
+  /* END GENERAL STUFF */
+
+
+
+  /* VIEW SSR STUFF */
+  .user-view-ssr-modal {
+    .panel:not(:last-child) {
+      th, span {
+        font-size: 9px;
+      }
+      td {
+        font-size: 8px;
+      }
+    }
+  }
+  /* END VIEW SSR STUFF */
+
+
+
+  /* CONSOLIDATED-REQUEST-SPECIFIC STUFF */
+  .full-calendar-modal {
+    .calendar-container {
+      th, span {
+        font-size: 7px;
+      }
+      td {
+        font-size: 8px;
+      }
+    }
+  }
+  /* END CONSOLIDATED-REQUEST-SPECIFIC STUFF */
+
+
+
+  /* PANELS */
```

Comment:
Have you looked into seeing if there is a way to include bootstrap externally here while within the scope of the print media, as in some sort of require line?